build/ProjectReunion-CI.yml

```diff
@@ -117,111 +122,115 @@ jobs:
-# is still being run in WindowsAppSDK-BuildMRT-Steps.yml.
-
-- job: StageAndPack
-  pool:
-    vmImage: 'windows-2022'
-  timeoutInMinutes: 120
-  dependsOn:
-    - Build
-    - BuildMRT
-  steps:
-  - task: DownloadBuildArtifacts@0
-    inputs:
-      artifactName: BuildOutput
-      downloadPath: '$(Build.SourcesDirectory)'
-
-  - task: PowerShell@2
-    name: StageFiles
-    inputs:
-      filePath: 'BuildAll.ps1'
-      arguments: -Platform "x86,x64,arm64" -Configuration "release" -AzureBuildStep "StageFiles"
-
-  - task: PowerShell@2
-    name: SetVersion
-    displayName: Update metapackage version
-    inputs:
-      targetType: 'inline'
-      script: |
-        $packageVersion = '$(version)'
-        $pipelineType = '$(PipelineType)'
-        $sourceBranchName = '$(Build.SourceBranchName)'
-        if ($sourceBranchName -eq 'main' -or $sourceBranchName -eq 'develop')
-        {
-          $packageVersion = $packageVersion + '.' + $sourceBranchName + '.' + $pipelineType
-        }
-        Write-Host "##vso[task.setvariable variable=packageVersion;]$packageVersion"
-        Write-Host $packageVersion
-        [xml]$publicNuspec = Get-Content -Path $(Build.SourcesDirectory)\build\NuSpecs\Microsoft.WindowsAppSDK.Foundation.nuspec
-        $publicNuspec.package.metadata.version = $packageVersion
-        Set-Content -Value $publicNuspec.OuterXml $(Build.SourcesDirectory)\build\NuSpecs\Microsoft.WindowsAppSDK.Foundation.nuspec
-  - task: PowerShell@2
-    name: PackNuget
-    inputs:
-      filePath: 'BuildAll.ps1'
-      arguments: -Platform "x64" -Configuration "release" -AzureBuildStep "PackNuget" -OutputDirectory "$(build.artifactStagingDirectory)\FullNuget" -PackageVersion "$(packageVersion)"
-
-  - task: BinSkim@3
-    inputs:
-      InputType: 'Basic'
-      Function: 'analyze'
-      AnalyzeTarget: '$(build.SourcesDirectory)\BuildOutput\FullNuget\*.dll;$(build.SourcesDirectory)\BuildOutput\FullNuget\*.exe'
-      AnalyzeVerbose: true
-
-  - task: PostAnalysis@1
-    inputs:
-      AllTools: false
-      APIScan: false
-      BinSkim: true
-      BinSkimBreakOn: 'Error'
-      CodesignValidation: false
-      CredScan: false
-      FortifySCA: false
-      FxCop: false
-      ModernCop: false
-      PoliCheck: false
-      RoslynAnalyzers: false
-      SDLNativeRules: false
-      Semmle: false
-      TSLint: false
-      ToolLogsNotFoundAction: 'Standard'
-
-  - task: PublishBuildArtifacts@1
-    displayName: 'Publish artifact: NugetContent'
-    inputs:
-      PathtoPublish: '$(Build.SourcesDirectory)\BuildOutput\FullNuget'
-      artifactName: 'NugetContent'
-
-  - task: PublishBuildArtifacts@1
-    displayName: 'Publish artifact: TransportPackage'
-    inputs:
-      PathtoPublish: '$(build.artifactStagingDirectory)\FullNuget'
-      artifactName: 'TransportPackage'
+# Public validation for BuildMRT will be disabled until Microsoft.Taef becomes public.
```

Comment:
Can you elaborate what you mean by "Microsoft.Taef becomes public"?
My understanding is that TAEF is already public:
https://learn.microsoft.com/en-us/windows-hardware/drivers/taef/

Reply:
The TAEF package is not available on Nuget.org yet